flink-kubernetes-operator/pom.xml

```diff
@@ -75,6 +75,12 @@ under the License.
             <version>${flink.version}</version>
         </dependency>
 
+        <dependency>
```

Comment:
We might need to update the notice file when introducing new dependency.

Reply:
The flink-core seems already included. 

```
- org.apache.flink:flink-core:1.14.4
```

Reply:
What I mean is the transitive dependencies? Do they have already included in the NOTICE file?

Reply:
Since the dependency of `flink-clients` already include the `flink-core`, So I remove the duplicated declaration here